deployments/utoronto/image/install.R

```diff
@@ -57,7 +57,7 @@ cran_packages <- c(
   "raster", "3.4-13",
   "rasterVis", "0.50.3",
   "RcppArmadillo", "0.10.1.2.0",
-  "rgdal", "1.5-27",
+  "rgdal", "1.5-25",
```

Comment:
We install R pkgs from https://packagemanager.rstudio.com and the latest available version there is [1.5-25](https://packagemanager.rstudio.com/client/#/repos/1/packages/rgdal)